server/src/main/java/org/opensearch/index/codec/CodecService.java

```diff
@@ -59,31 +62,29 @@ public class CodecService {
 
     private final Map<String, Codec> codecs;
 
-    public static final String DEFAULT_CODEC = "default";
```

Comment:
Removing public constants could break things for others (fine for 3.x but should be kept for 2.x backport I think).

Reply:
reinstated